jfr-daemon/build.gradle.kts

```diff
@@ -15,26 +16,20 @@ java {
     disableAutoTargetJvm()
 }
 
-dependencies {
-    api(project(":jfr-mappers"))
-    api("org.slf4j:slf4j-api:${Versions.slf4j}")
-    api("org.apache.logging.log4j:log4j-slf4j-impl:${Versions.log4j}")
-    api("org.apache.logging.log4j:log4j-core:${Versions.log4j}")
-    api("com.newrelic.telemetry:telemetry-http-java11:${Versions.newRelicTelemetry}")
-    implementation("com.google.code.gson:gson:${Versions.gson}")
-}
-
-tasks.shadowJar {
-    archiveClassifier.set("")
-    manifest {
-        attributes(
-                "Main-Class" to "com.newrelic.jfr.daemon.JFRDaemon",
-                "Implementation-Version" to project.version
-        )
```

Comment:
So this changes the behavior of the daemon in a way that isn't documented. The project still publishes a jar, but its no longer a fat jar and doesn't function as described here: https://docs.newrelic.com/docs/agents/java-agent/features/real-time-profiling-java-using-jfr-metrics

The publish task will still run, but its just a regular jar so its functionally broken.